### DISCUSSION TITLE
OQS Provider not loading for android device

### DISCUSSION THREAD

**Discussion by DeekshantDewari** [open-quantum-safe/oqs-provider#513] [Q&A]
Hi, 
Can anyone help me regarding setting my OQS provider ,I have compiled my openssl OQS provider and liboqs for my android device and put .so file inside my jniLibs directory I have updated the openssl.cnf file and put it in my assets folder of my project directory . I'm able to successfully initialise openssl but OQS provider is not getting access. If anyone have some suggestion please do let me know.
Thanks in advance

**Comment by baentsch** [comment]
Sorry, I have no android experience -- and it seems no-one else in the community does either.
